sql/moz-fx-data-shared-prod/telemetry_dev_cycle_external/expirements_stats_v1/query.py

```diff
@@ -0,0 +1,153 @@
+    bigquery.SchemaField("end_date", "DATE"),
+    bigquery.SchemaField("has_config", "BOOLEAN"),
+]
+DEFAULT_PROJECT_ID = "moz-fx-data-shared-prod"
+DEFAULT_DATASET_ID = "telemetry_dev_cycle_external"
+DEFAULT_TABLE_NAME = "experiments_stats_v1"
+
+
+def parse_unix_datetime_to_string(unix_string):
+    """Parse unix_string with milliseconds to date string."""
+    if not unix_string:
+        return None
+    return datetime.datetime.fromtimestamp(int(unix_string) // 1000).strftime(
+        "%Y-%m-%d"
+    )
+
+
+def get_api_response(url):
+    """Get json of response if the requests to the API was successful."""
+    response = requests.get(url)
+    response.raise_for_status()
+    return response.json()
+
+
+def store_data_in_bigquery(data, schema, destination_project, destination_table_id):
+    """Upload data to Bigquery in a single, non partitioned table."""
+    client = bigquery.Client(project=destination_project)
+
+    job_config = bigquery.LoadJobConfig(
+        create_disposition="CREATE_IF_NEEDED",
+        schema=schema,
+        source_format=bigquery.SourceFormat.NEWLINE_DELIMITED_JSON,
+        write_disposition="WRITE_TRUNCATE",
+    )
+
+    load_job = client.load_table_from_json(
+        data, destination_table_id, location="US", job_config=job_config
+    )
+    load_job.result()
+    stored_table = client.get_table(destination_table_id)
+    logging.info(f"Loaded {stored_table.num_rows} rows into {destination_table_id}.")
+
+
+def download_experiments_v1(url):
+    """Download experiment data from API v1 and parse it."""
+    experiments_v1 = []
+    experiments = get_api_response(f"{url}/api/v1/experiments")
+    for experiment in experiments:
+        if experiment["status"] == "Draft":
+            continue
+        experiments_v1.append(
+            {
+                "slug": experiment["slug"],
+                "start_date": parse_unix_datetime_to_string(experiment["start_date"]),
+                "enrollment_end_date": None,
+                "end_date": parse_unix_datetime_to_string(experiment["end_date"]),
+            }
+        )
+    return experiments_v1
+
+
+def download_experiments_v6(url):
+    """Download experiment data from API v6 and parse it."""
+    experiments_v6 = []
+    experiments = get_api_response(f"{url}/api/v6/experiments")
+    for experiment in experiments:
+        experiments_v6.append(
+            {
+                "slug": experiment["slug"],
+                "start_date": experiment["startDate"],
+                "enrollment_end_date": experiment["enrollmentEndDate"],
+                "end_date": experiment["endDate"],
+            }
+        )
+    return experiments_v6
+
+
+def download_metric_hub_files(url):
+    """Download metric hub files from github."""
+    metric_files = {}
+    files = get_api_response(url)
+    for file in files["payload"]["tree"]["items"]:
+        if file["contentType"] != "file":
+            continue
+        slug = file["name"].replace(".toml", "")
```

Comment:
```suggestion
        slug = file["name"].removesuffix(".toml")
```

https://docs.python.org/3/library/stdtypes.html#str.removesuffix